doc-Appliance_Hardening_Guide/cfme/master.adoc

```diff
@@ -0,0 +1,30 @@
+= Appliance Hardening Guide
+:vernum: 4.1
+:doctype: book
+:toc: left
+:toclevels: 3
+:numbered!:
+:experimental:
+
+// Custom attributes
+include::common/attributes/cfme.adoc[]
+
+:cfme:
+
+:numbered:
+:leveloffset: 1
+include::topics/Introduction.adoc[]
```

Comment:
There's nothing functionally wrong with this, but I just want to note that including a group of topics together in a single document is not consistent with the "topic-oriented documentation" model, which has been adopted by other docs teams in Red Hat.